man/systemd-sleep.conf.xml

```diff
@@ -113,6 +113,24 @@
     <filename>sleep.conf.d</filename> file:</para>
 
     <variablelist class='systemd-directives'>
+      <varlistentry>
+        <term><varname>AllowSuspend=</varname></term>
+        <term><varname>AllowHibernation=</varname></term>
+        <term><varname>AllowSuspendThenHibernate=</varname></term>
+        <term><varname>AllowHybridSleep=</varname></term>
```

Comment:
Is "Allow" the right word for this? That suggests an access control concept, but this is really not so much about that. Hence maybe call this "OfferSuspend=" or something like that? "ProvideSuspend=" or something like that?

Reply:
"Offer" on the the other hand suggests that the thing will be offered unconditionally. I still think "Allow" is best of the three options because it *is* about access control, not for security, but for this coding acting as a filter what is advertised.